pygeos/set_operations.py

```diff
@@ -16,45 +16,87 @@
 ]
 
 @multithreading_enabled
-def difference(a, b, **kwargs):
+def difference(a, b, grid_size=None, **kwargs):
     """Returns the part of geometry A that does not intersect with geometry B.
 
+    If grid_size is nonzero, input coordinates will be snapped to a precision grid of that
```

Comment:
Can you limit the line length here to 80? (black doesn't fix this in a docstring, since this one is even longer as 88, and in addition for docstring readability (and eg how it is displayed in jupyter popups) it is good to even limit to 80 for docstrings)